### PR TITLE
Issue #13213: Remove '//ok' comments from blocks package

### PR DESCRIPTION
**Pull request by relentless-pursuit**
part of https://github.com/checkstyle/checkstyle/issues/13213

link to documentation: https://checkstyle.sourceforge.io/checks/blocks/index.html

```
%  grep -A 1 'UnnecessaryOkComment' checkstyle-input-suppressions.xml | grep -B 1 'blocks'| sed '/^--$/d' | grep -c 'UnnecessaryOkComment'
473

%  grep -A 1 'UnnecessaryOkComment' checkstyle-input-suppressions.xml | grep -B 1 'blocks'| sed '/^--$/d' | grep -c 'UnnecessaryOkComment'
0

```

### CODE REVIEW

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/leftcurly/InputLeftCurlyCommentBeforeLeftCurly.java [hunk 1]
```diff
@@ -21,10 +21,10 @@
 public class InputLeftCurlyCommentBeforeLeftCurly {
 
     void method1 ()
-    /* violation not reported */ { } // ok until #11410
+    /* violation not reported */ { }
 
     InputLeftCurlyCommentBeforeLeftCurly()
-    /* I am comment */ { // ok until #11410
+    /* I am comment */ {
```
[the remark this answers is not in the document]
Comment (relentless-pursuit) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/leftcurly/InputLeftCurlyCommentBeforeLeftCurly.java]: done

**Review by romani — APPROVED** [review]
Ok to merge.

Thanks a lot for update

**Review by nrmancuso — APPROVED** [review]

**Comment by nrmancuso** [comment]
Ok to merge as CI passes

Looks like GitHub actions did not run on the last push.

**Pull request merged**